### PR TITLE
CU-2fdkx0f - Refactor Vamm Associated Types

### PR DESCRIPTION
**Pull request by 0xangelo**
## Issue
https://app.clickup.com/t/2fdkx0f

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2fdkx0f Refactor Vamm Associated Types](https://app.clickup.com/t/2fdkx0f)

**Review by Cardosaum — APPROVED** [review]
Great!

**Comment by 0xangelo** [comment]
@Cardosaum now it's your turn :)

File: frame/clearing-house/src/math.rs [hunk 1]
```diff
@@ -99,35 +99,18 @@ where
...
 }
 
-// The following types are markers to disambiguate the two implementations of IntoBalance below.
-// Since traits can overlap, the 'where' clauses below are not enough to tell the compiler which
-// implementation of the trait it should use for a particular type
-pub struct Int;
-pub struct FixedPoint;
-
-pub trait IntoBalance<Balance, Marker> {
+pub trait IntoBalance<Balance> {
```
Comment (Cardosaum) [inline frame/clearing-house/src/math.rs:102]: nice that you were able to remove this hack in a really clean way! :+1:

**Review by Cardosaum — APPROVED** [review]
LGTM! :100:

**Review by 0xangelo — COMMENTED** [review]
LGTM

File: frame/clearing-house/src/math.rs (hunk 1 above)
Comment (0xangelo) [reply frame/clearing-house/src/math.rs:102]: Yeah, I was relieved to do it 😅

**Pull request merged**